tests/_support/Step/Acceptance/Administrator.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected vipgoci-phpcs-scan-d2okzw.php, but found vipgoci-phpcs-scan-D2OKzW.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).